src/app/components/form/FormDelete.vue

```diff
@@ -75,15 +77,33 @@ const rules = {
   password: VALIDATION_PASSWORD(),
 }
 const v$ = useVuelidate(rules, form)
+
+watch(
+  () => api.value.errors,
+  (current) => {
+    modelError.value = current?.length
+      ? new Error(
+          getCombinedErrorMessages(current, {
+            postgres23503: t('postgres23503'),
+            postgres28P01: t('postgres28P01'),
+          })[0],
+        )
+      : undefined
+  },
+)
 </script>
 
 <i18n lang="yaml">
 de:
   deletion: '{item} löschen'
   passwordAccount: Konto-Passwort
+  postgres23503: Dir gehören noch Daten! Lösche erst all deine Veranstaltungen, Kontakte und Bilder.
+  postgres28P01: Passwort falsch! Überprüfe, ob du alles richtig geschrieben hast.
```

Comment:
The generic delete form shouldn't know about postgres error messages which belong to a certain mutation. Move them up to the component which defines the mutation.

Reply:
Moved, you can resolve when checked